test/remote/gateways/remote_usa_epay_transaction_test.rb

```diff
@@ -22,6 +22,20 @@ def test_successful_purchase_with_track_data
     assert_success response
   end
 
+  def test_successful_authorization_with_manual_entry
+    @credit_card.manual_entry = true
```

Comment:
`@creditcard` is used 31 times in the test directory vs 2893 times for `@credit_card` so i think I will change it.